### PR TITLE
Bug 2032732: Fix styling conflicts stemming from console wide changes

### PR DESCRIPTION
**Pull request by rexagod**
Fix styling conflicts that stemmed from recent console wide changes that
got in.
***
<details>
<summary>Screenshots</summary>

Screenshot from 2021-12-15 11-50-01

Screenshot from 2021-12-15 11-50-25

Screenshot from 2021-12-15 11-49-49

</details>

### CODE REVIEW

**Comment by rexagod** [comment]
/hold

Adding screenshots.

**Comment by SanjalKatiyar** [comment]
SS LGTM...
however unrelated to this PR: https://user-images.githubusercontent.com/33557095/146134101-2db2f549-d4dd-4ab8-a773-c74e47f6683f.png
there are differences in the dropdowns, example: dropdown on used-capacity card and the one on utilization card. I suppose that is because we are using different components for both of them. Not sure if we need to fix it right now.
@bipuladh @rexagod wdyt ?

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/utilization-card.tsx [hunk 1]
```diff
@@ -51,49 +50,51 @@ const UtilizationCard: React.FC = () => {
...
-        <PrometheusUtilizationItem
-          title={t('ceph-storage-plugin~Recovery')}
-          utilizationQuery={
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_RECOVERY_RATE_QUERY]
-          }
-          humanizeValue={humanizeDecimalBytesPerSec}
-        />
-      </UtilizationBody>
+      <div className="co-utilization-card__body">
+        <Grid>
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/utilization-card.tsx:54]:
```suggestion
        <Grid className="co-utilization-card__body">
```
WhaT about doing this?

**Comment by bipuladh** [comment]
> SS LGTM... however unrelated to this PR: https://user-images.githubusercontent.com/33557095/146134101-2db2f549-d4dd-4ab8-a773-c74e47f6683f.png there are differences in the dropdowns, example: dropdown on used-capacity card and the one on utilization card. I suppose that is because we are using different components for both of them. Not sure if we need to fix it right now. @bipuladh @rexagod wdyt ?

Let's fix it in a different PR. We might have to adjust the style difference using CSS.

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/utilization-card.tsx (hunk 1 above)
Comment (rexagod) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/utilization-card.tsx]: Changed.

**Pull request merged**